packages/forma-36-react-components/src/components/Table/TableBody/TableBody.tsx

```diff
@@ -1,9 +1,14 @@
 import React, { Component } from 'react';
 
+import SkeletonTable from '../../Skeleton/SkeletonTable';
+
 export type TableBodyProps = {
   className?: string;
   style?: React.CSSProperties;
   testId?: string;
+  loading?: boolean;
```

Comment:
We've got some naming conventions we're trying to stick to with components - https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#component-principles - so I recommend changing to the following:

```suggestion
  isLoading?: boolean;
```

Reply:
Cool! Thanks for the link too 😄